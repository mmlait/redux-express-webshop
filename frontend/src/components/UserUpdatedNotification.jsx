import React from 'react';
import styled from 'styled-components';
import colors from '../colors';

const NotificationWrapper = styled.div`
  background-color: ${colors.greenModalBg};
  color: ${colors.fontLight};
  font-size: 1rem;
  border-radius: 3px;
  right: 5px;
  position: fixed;
  z-index: 1;
  margin-top: 55px;
  top: 0;
  padding: 0 10px;
  transform: translateX(100%);
  -webkit-animation: anim 4s;
  animation: anim 4s;
  @-webkit-keyframes anim {
    0% {
      transform: translateX(100%);
    }
    14.28% {
      transform: translateX(0);
    }
    85.71% {
      transform: translateX(0);
    }
    100% {
      transform: translateX(100%);
    }
  }
  @keyframes anim {
    0% {
      transform: translateX(100%);
    }
    14.28% {
      transform: translateX(0);
    }
    85.71% {
      transform: translateX(0);
    }
    100% {
      transform: translateX(100%);
    }
  }
`

const UserUpdatedNotification = () => {
  return (
    <NotificationWrapper id="userUpdatedNotification">
      <p>Your information has been updated!</p>
    </NotificationWrapper>
  );
}

export default UserUpdatedNotification;
