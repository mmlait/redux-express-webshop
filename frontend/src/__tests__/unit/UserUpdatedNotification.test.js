import React from 'react';
import { create } from 'react-test-renderer'
import UserUpdatedNotification from '../../components/UserUpdatedNotification';

describe('UserUpdatedNotification component', () => {
    it('should render', () => {
    let tree = create(<UserUpdatedNotification />)
    expect(tree.toJSON()).toMatchSnapshot();
  })
})