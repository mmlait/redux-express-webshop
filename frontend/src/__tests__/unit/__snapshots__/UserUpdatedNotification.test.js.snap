// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserUpdatedNotification component should render 1`] = `
<div
  className="sc-bdVaJa gtzOcq"
  id="userUpdatedNotification"
>
  <p>
    Your information has been updated!
  </p>
</div>
`;
